shared/actions/chat2/index.js

```diff
@@ -2015,7 +2015,6 @@ function* chat2Saga(): Saga.SagaGenerator<any, any> {
     )
   } else {
     yield Saga.safeTakeEveryPure(Chat2Gen.desktopNotification, desktopNotify)
-    yield Saga.safeTakeEveryPure(Chat2Gen.previewConversation, navigateToThread)
```

Comment:
Seems clearer to have the previewConversation handler navigate when it needs to (i.e., after it selects the pending conversation)

Reply:
lets discuss. I think the relationships between the sagas should be that we have 
takeEvery => sideeffect instead of
takeEvery => handler-which-spawns-a-bunch-of-actions
more fan-in than fan-out. anyways, easier to chat over video

Reply:
Okay, fixed -- `changePendingMode` was already looking at `PreviewConversationPayload`, so moved the logic there